app/models/institution.rb

```diff
@@ -230,31 +237,37 @@ def self.autocomplete(search_term, limit = 6)
 
   # Finds exact-matching facility_code or partial-matching school and city names
   #
-  scope :search, lambda { |search_term, include_address = false, fuzzy_search = false|
+  scope :search, lambda { |search_term, include_address = false, fuzzy_search_flag = false,
+      use_fuzzy_search = false
+  |
     return if search_term.blank?
 
-    clause = ['facility_code = :facility_code']
+    clause = ['facility_code = :upper_search_term']
 
-    if fuzzy_search
-      clause << 'SIMILARITY(institution, :search_term) > :name_threshold'
-      clause << 'SIMILARITY(city, :search_term) > :city_threshold'
-      clause << 'zip = :search_term'
+    if fuzzy_search_flag
+      if use_fuzzy_search
```

Comment:
I know I'm not very familiar with gibct, but what's the difference between `app/models/institution.rb` & `use_fuzzy_search`?

Reply:
`fuzzy_search_flag` comes from a Feature Flag. It is based on the status of https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/features.yml#158 when used in the frontend at https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/gi/containers/SearchPage.jsx

`use_fuzzy_search?` is an additional check that checks if the counts of certain criteria combined is higher than 0 and causes it to use more effective and efficient SQL when searching

Reply:
the status of the feature flag is passed down as a parameter to gibct because feature flags are not a part of this repo as they are in vets-api